pubspec.yaml

```
@@ -14,4 +14,5 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_lints: ^2.0.0
\ No newline at end of file
+  flutter_lints: ^2.0.0
+  path: ^1.9.0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)